### DISCUSSION TITLE
Error importing pymeshlab

### DISCUSSION THREAD

**Discussion by wikyburns** [cnr-isti-vclab/PyMeshLab#65] [Q&A] [answered]
Hi guys, I never used python for any project, it's simple but i'm having this issue when i'm trying to import pymeshlab.

Actually i have used on my windows machine and everything works fine, but not in ubuntu 20.04.

![imagen](https://user-images.githubusercontent.com/11525781/109049514-ac42bb00-76d8-11eb-93a3-27115bfe2627.png)

any steps to follow?

really thanks!

**Comment by alemuntoni** [comment, marked as answer]
Hi, have you tried to install the packages listed in this question:
https://stackoverflow.com/questions/55313610/importerror-libgl-so-1-cannot-open-shared-object-file-no-such-file-or-directo

**Reply by wikyburns** [reply]
Hi @alemuntoni thanks for the fast reply! and yes, it works!

Really thanks!
